glean-core/src/lib.rs

```diff
@@ -694,6 +694,15 @@ pub fn glean_test_get_experiment_data(experiment_id: String) -> Option<RecordedE
     core::with_glean(|glean| glean.test_get_experiment_data(experiment_id.to_owned()))
 }
 
+/// Sets a remote configuration for the metrics' disabled property
+///
+/// See [`core::Glean::set_metrics_disabled_config`].
+pub fn glean_set_metrics_disabled_config(json: String) {
+    launch_with_glean(|glean| {
+        _ = glean.set_metrics_disabled_config(json);
```

Comment:
Not sure what to do with the error here, probably should allow it to bubble up across the FFI